src/main/java/org/opentripplanner/api/resource/GraphPathToTripPlanConverter.java

```diff
@@ -213,10 +213,7 @@ public static Itinerary generateItinerary(GraphPath path, boolean showIntermedia
 
         itinerary.walkDistance = lastState.getWalkDistance();
 
-        itinerary.transfers = lastState.getNumBoardings();
-        if (itinerary.transfers > 0) {
-            itinerary.transfers--;
-        }
+        itinerary.transfers = 0;
```

Comment:
Javadoc for this method (and also the method `generatePlan` that calls it, and perhaps the other methods it calls) should be updated to clarify that it now only produces TripPlans for non-transit itineraries, and non-transit legs of itineraries containing transit, with the whole transit itinerary produced by ItineraryMapper.